easyaudit/signals/request_signals.py

```diff
@@ -50,7 +50,7 @@ def request_started_handler(sender, environ, **kwargs):
         url=environ['PATH_INFO'],
         method=environ['REQUEST_METHOD'],
         query_string=environ['QUERY_STRING'],
-        user=user,
+        user_id=getattr(user, 'id', None),
```

Comment:
Is there a reason why you changed `user` to `user_id` (and also, `content_type` to `content_type_id`, etc.) when creating Event objects?

Reply:
Yes. Mostly for being explicit because we only have IDs technically supported on this code change. So I do think it would fail (on my project, since the db has been split).